assets/src/components/resource/editors/dragndrop/handlers/drop.ts

```diff
@@ -30,7 +28,7 @@ export const dropHandler =
 
         const sourceIndex = content.findIndex((k) => k.id === droppedContent.id);
 
-        if (sourceIndex === []) {
+        if (sourceIndex.length === 0) {
```

Comment:
Strict comparison of a list to [] will never be true, since they are different objects. Best way to check if list is empty is to use `length`